extension/packages/nextjs/components/Header.tsx.args.mjs

```diff
@@ -0,0 +1,6 @@
+export const menuIconImports = `import { MagnifyingGlassIcon } from "@heroicons/react/24/outline";`;
+export const menuObjects = `{
+  label: "Greetings",
+  href: "/greetings",
```

Comment:
What do you think if we do?

"Ponder"
"/ponder-greetings"

And then explaining more in the page as I said in the previous comment.

Reply:
Perfect! Thanks for the feedback!